index.html

```diff
@@ -781,22 +863,98 @@ <h5 class="noselect nocookie">
         }, 1);
       };
 
+
+      //from where?
+
       var QueryString = window.location.search;
       var urlParams = new URLSearchParams(QueryString);
       var from = urlParams.get("from");
 
       if (from) {
-        fetch("https://terrainstamper.biddydev.com/api/users/userfrom", {
+        fetch("https://backend.atlasterrain.com/api/users/userfrom", {
           body: JSON.stringify({ from }),
           method: "POST",
           headers: new Headers({ "content-type": "application/json" }),
+        }).then(() => {
+          const builtUrl = window.location.host;
+          window.location.replace(window.location.origin);
+        }).catch(console.error);
+      }
+
+
+      //mail box yeee
+
+      var mail = document.getElementById("mail")
+      var submit = document.getElementById("submit")
+      var mailboxinput = document.getElementById("mailboxinput")
+      var mailboxerror = document.getElementById("mailboxerror")

```

Comment:
```suggestion
      const mailboxerror = document.getElementById("mailboxerror")
```

the scope of the `var` keyword is terrible in JS. It's best to use either `const` or `let`. 
I use const for everything unless I KNOW the value will be changed later. Then I use let.